easybuild/easyconfigs/n/nd2reader/nd2reader-3.0.6-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'Bundle'
+
+name = 'nd2reader'
+version = '3.0.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://soft-matter.github.io/pims'
+description = "PIMS is a lazy-loading interface to sequential data with numpy-like slicing."
```

Comment:
@boegel - this homepage and description don't look right. Shouldn't they be taken from https://github.com/soft-matter/pims_nd2?

Reply:
Ugh, copy-paste mistake, will fix, thanks.